tests/test_admin.py

```diff
@@ -2708,6 +2708,71 @@ def test_extended_version_change_list_actions_burger_menu_available(self):
         self.assertContains(response, "versioning_static_url_prefix")
         self.assertTrue(soup.find("script", src=re.compile("djangocms_versioning/js/actions.js")))
 
+    def test_extended_version_change_list_author_ordering(self):
+        """
+        The author is sortable by username in both ascending and descending order
+        """
+        # Create a series of users, so we can order them alphabetically by username!
+        user_first = factories.UserFactory(username="A Username Capitalised")
+        user_first_lower = factories.UserFactory(username="a username lower")
+        user_middle = factories.UserFactory(username="Middle Username Capitalised")
+        user_middle_lower = factories.UserFactory(username="middle username lower")
+        user_last = factories.UserFactory(username="Z Username Capitalised")
+        user_last_lower = factories.UserFactory(username="z username lower")
+        # Create some pollcontent and their corresponding versions, and polls!
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_first
+        )
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_first_lower
+        )
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_middle
+        )
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_middle_lower
+        )
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_last,
+        )
+        factories.PollVersionFactory(
+            content=factories.PollContentFactory(language="en"),
+            created_by=user_last_lower,
+        )
+
+        with self.login_user_context(self.get_superuser()):
+            base_url = self.get_admin_url(PollContent, "changelist")
+            base_url += "?o=3"
```

Comment:
May need to add one thing to this test to ensure that the index of the 3rd field matches the author field, this will break if anyone adds a new field and someone fixing the test may not know why it's failing. Maintenance of tests is how tests get "patched" and broken forever. The core cms suffers badly from this where half the suite doesn't actually test what it was intended to test anymore. 

To bullet proof it you could get the list display of the admin model and find the index by the field name, that way any changes to fields won't require a manual change.

Reply:
```
from django.contrib import admin

# Find the index of the author field to sort by the correct field
modeladmin = admin_site._registry[PollContent]
admin_field_list = modeladmin.get_list_display(request)
author_index = admin_field_list.index("get_author")
```

I'm seeing this as index 1 locally with PollContent so not sure if '3' is correct. 3 would be version state: 
<img width="1792" alt="Screenshot 2022-06-08 at 16 04 06" src="https://user-images.githubusercontent.com/12543977/172651612-afbd4eba-9a5f-4e06-a779-702fd6c0402d.png">